import-service/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "import-service",
+  "version": "1.0.0",
+  "description": "Serverless webpack example using Typescript",
+  "main": "handler.js",
+  "scripts": {
+    "test": "jest"
+  },
+  "dependencies": {
+    "aws-sdk": "^2.792.0",
```

Comment:
It's better to add aws-sdk as devDependency. Lambda runtime environment already has aws-sdk methods.